traits/ctraits.c

```diff
@@ -4617,10 +4749,25 @@ static PyMethodDef trait_methods[] = {
 +----------------------------------------------------------------------------*/
 
 static PyGetSetDef trait_properties[] = {
-        { "__dict__",     (getter) get_trait_dict,    (setter) set_trait_dict },
-        { "handler",      (getter) get_trait_handler, (setter) set_trait_handler },
-        { "post_setattr", (getter) get_trait_post_setattr,
-                      (setter) set_trait_post_setattr },
+        { "__dict__",       (getter) get_trait_dict,    (setter) set_trait_dict },
+        { "handler",        (getter) get_trait_handler, (setter) set_trait_handler },
+        { "post_setattr",   (getter) get_trait_post_setattr,
+                            (setter) set_trait_post_setattr },
+        { "property_flag",  (getter) get_trait_property_flag, NULL },
```

Comment:
Please add docstrings for the new properties. (At some point, we should also add docstrings for the existing ones too, but not as part of this PR.)